### DISCUSSION TITLE
Force feedback not working in BeamNG Logitech G920

### DISCUSSION THREAD

**Discussion by TheRealMSL** [berarma/oversteer#176] [Q&A]
I have a Logitech G920 and attempting to play beamng but I have zero force feedback. However, force feedback is working in assetto corsa. 

Is there any fix for this?

**Comment by leillo1975** [comment]
You must go to FFB settings and set the Update Type to Full:

https://www.beamng.com/threads/workaround-to-get-ffb-to-work-on-linux-proton.93556/

This is not a Oversteer related problem

**Comment by TheRealMSL** [comment]
Setting the force feedback to "Full" makes my wheel move left to right randomly making it impossible for me to use the wheel and the Wheel UI app proves that the wheel is not following force feedback at all.
